lib/stack_master/parameter_resolvers/parameter_store.rb

```diff
@@ -1,16 +1,23 @@
 module StackMaster
   module ParameterResolvers
     class ParameterStore < Resolver
+
+      SecretNotFound = Class.new(StandardError)
+
       def initialize(config, stack_definition)
         @config = config
         @stack_definition = stack_definition
       end
 
       def resolve(value)
-        resp = ssm.get_parameter(
-          name: value,
-          with_decryption: true
-        )
+        begin
+          resp = ssm.get_parameter(
+            name: value,
+            with_decryption: true
```

Comment:
Does this fail if the parameter is not encrypted?

Reply:
Nope, it's ignored for anything except a SecureString